src/about/About.js

```diff
@@ -11,38 +11,43 @@ import {
 
 let styles = {}
 
-class About extends Component {
-  render () {
-    return (
-      <ScrollView style={{backgroundColor: 'white'}}>
-        <View style={styles.headingContainer}>
-          <Image source={require('../images/dailydrip.png')} />
-          <Text style={styles.heading}>About</Text>
-        </View>
-        <View style={styles.container}>
-        <Card
-            title='Contribute with us'>
-            <Text style={{marginBottom: 10}}>
-              This project is totally open source by DailyDrip.
-            </Text>
-            <Button
-              icon={{name: 'code'}}
-              backgroundColor='#03A9F4'
-              buttonStyle={{borderRadius: 0, marginLeft: 0, marginRight: 0, marginBottom: 0}}
-              title='Check the code' />
-          </Card>
-        </View>
-      </ScrollView>
-    )
-  }
+const About = () => {
+  return (
+    <ScrollView style={{backgroundColor: 'white'}}>
+      <View style={styles.headingContainer}>
+        <Image source={require('../images/dailydrip.png')} />
+        <Text style={styles.heading}>About</Text>
+      </View>
+      <View style={styles.container}>
+      <Card
+          title='Contribute with us'>
+          <Text style={{marginBottom: 10}}>
+            This project is totally open source by DailyDrip.
+          </Text>
+          <Button
+            icon={{name: 'code'}}
+            backgroundColor='#03A9F4'
+            onPress={() => Linking.openURL('https://github.com/dailydrip/react-native-docs').catch(err => console.error('An error occurred', err)) }
+            buttonStyle={{borderRadius: 0, marginLeft: 0, marginRight: 0, marginBottom: 0}}
+            title='Check the code' />
+        </Card>
+      </View>
+    </ScrollView>
+  )
 }
 
+About.navigationOptions = {
+  tabBar: {
+    icon: () =>(
+      <Icon style={{paddingBottom: 4 }} name='question-answer' size={26} />
```

Comment:
question-answer is the font-awesome + Icons: https://oblador.github.io/react-native-vector-icons/ (search for question-answer).